gandalf.js

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.